test/run_test.py

```diff
@@ -1318,14 +1320,17 @@ def main():
     )
     os.makedirs(REPO_ROOT / "test" / "test-reports", exist_ok=True)
 
-    def success_callback(err_message):
+    def handle_error_messages(err_message):
         if err_message is None:
-            return True
```

Comment:
Note that nothing actually read the return value of this method before, so it's okay to invert it for better code ergonomics.